packages/markdown-utils/index.ts

```diff
@@ -1,10 +1,28 @@
-import { allPlugins } from '@hashicorp/remark-plugins'
```

Comment:
`remark-plugins` had an `allPlugins` export, `platform-remark-plugins` does not. With an eye on trying to better compose these utilities, rather than inheriting default behaviour, I think it might be preferable to drop `allPlugins` (and I think this reflects decisions made when migrating `remark-plugins`?).

Have added a temporary backfill here since that felt like a good first step forward (while keeping the scope of this PR reasonable).

Reply:
This should be available at `@hashicorp/platform-remark-plugins/all`: https://github.com/hashicorp/web-platform-packages/blob/main/packages/remark-plugins/all.js